components/articles/kiera-king.tsx

```diff
@@ -0,0 +1,52 @@
+import {Title} from "../infoColumns/infoColumns.styles";
+import {Body, SubTitle} from "../../styles/typography";
+import {MaterialCard} from "../../app/components/elements";
+import {colors} from "../../styles/colors";
+import Link from "next/link";
+
+
+export default function KieraKingAwardDetails(): JSX.Element {
+  return (
+      <MaterialCard column noAnimate background={ colors.ivory }>
+        <Title margins="2vw 8rem 0 8rem" line={1.2} align={ "left" }>
+          Chicago International Model United Nations: Kiera King Delegate Award
+      </Title>
+      <Body margins="2rem 8rem 0.5rem 8rem" mobMargins="1rem 1rem 0 1rem"
+            line={ 1.4 } align={ "justify" } size="1.5rem">
```

Comment:
do you have prettier installed / do we have a prettier file in this repo? the indenting inside of tags looks off like the tag wraps and indents but the code under it outside of the tag is not indented